package.json

```diff
@@ -16,7 +16,7 @@
     "build": "yarn clean && turbo run build",
     "clean": "yarn install && turbo run clean && rimraf .turbo && rimraf *.tsbuildinfo",
     "clean:full": "yarn install && turbo run clean:full && rimraf .turbo && rimraf *.tsbuildinfo && rimraf node_modules",
-    "dev": "turbo run dev",
+    "dev": "yarn build && turbo run dev",
```

Comment:
1. is `yarn build` necessary?

2. it appears the `yarn build` is need for `client` only, better structure to limit this to `client` only
e.g. in `apps/client/package.json`, do `"dev": "yarn build && DISABLE_ESLINT_PLUGIN=true BROWSER=none PORT=3001 vite",`